src/assisted_test_infra/test_infra/helper_classes/cluster.py

```diff
@@ -136,6 +140,19 @@ def _create(self) -> str:
         self._config.cluster_id = cluster.id
         return cluster.id
 
+    def get_olm_operators(self) -> List:
```

Comment:
```suggestion
    def get_olm_operators(self) -> List[str]:
```
`List` isn't complete without the type held inside